### DISCUSSION TITLE
feature request: Dark mode for options page

### DISCUSSION THREAD

**Discussion by amithm7** [jordwest/news-feed-eradicator#392] [Ideas]
light theme is blinding at night.

Can send a PR if something like this would be accepted:

![image](https://user-images.githubusercontent.com/7036770/137723272-f8658b7c-f6ad-471f-a98d-a08e8a52daa1.png)

**Comment by ghost** [comment]
yes please

**Comment by amithm7** [comment]
@jordwest What is there to even discuss here? If you won't accept a PR for this, you could have just closed this.

**Comment by imurodl** [comment]
still there is no dark theme

**Comment by jordwest** [comment]
> @jordwest What is there to even discuss here? If you won't accept a PR for this, you could have just closed this.

I'm converting all issues that are ideas and suggestions across to discussions, this one was just one of many.

Dark theme is coming in v3
